temporalio/ui
# DT-1017 - support unique workflow table columns per namespace

## Description & motivation 💭 
Previously on the workflows list page, we would only fetch global search attributes to render the available table columns for the workflows summary table. With the change introduced in #1414, we fetch namespace specific search attributes any time the namespace path parameter changes. This allows us to let users configure their table columns on a per-namespace basis.

### Screenshots (if applicable) 📸 

https://github.com/temporalio/ui/assets/11775628/91218af7-bba4-4d36-9fd9-4bd368cc336e


### Design Considerations 🎨 
N/a
## Testing 🧪 

### How was this tested 👻 
This is covered by existing e2e and unit tests, as well as manual testing
- [x] Existing E2E tests
- [x] Existing Unit tests 

### Steps for others to test: 🚶🏽‍♂️🚶🏽‍♀️ 
- On the workflows list page, add/remove/reorder/pin/unpin columns on one namespace and ensure that the columns do not change for a second namespace and vice versa.
- Ensure only custom search attributes for a given namespace show in the side panel for managing columns.
## Checklists
N/a
### Draft Checklist 

### Merge Checklist 

### Issue(s) closed 

## Docs
N/a
### Any docs updates needed?
